code/game/objects/items/weapons/switchtool.dm

```diff
@@ -59,6 +62,9 @@
 		choose_deploy(user)
 
 /obj/item/weapon/switchtool/attackby(var/obj/item/used_item, mob/user)
+	if(istype(used_item, /obj/item/weapon/screwdriver) && no_remove)
```

Comment:
I try to account for everyone, also it says nowhere else that you cant remove modules or for the matter put more in. Stupid people do exist and I felt a sign that shows you clearly can't do something and not a bug was needed.